packages/opencensus-exporter-prometheus/src/prometheus-stats.ts

```diff
@@ -209,8 +220,11 @@ export class PrometheusStatsExporter implements StatsEventListener {
    * @param view View used to get the DistributionData
    * @param tags Tags used to get the DistributionData
    */
-  private getBoundaries(view: View, tags: Tags): number[] {
-    const data = view.getSnapshot(tags) as DistributionData;
+  private getBoundaries(view: View, labels: labelValues): number[] {
+    const tagValues = Object.keys(labels).map((label) => {
+      return {value: String(labels[label])};
+    });
+    const data = view.getSnapshot(tagValues) as DistributionData;
```

Comment:
Why is the `as DistributionData` needed?

Reply:
Only DistributionData has buckets property + ```getSnapshot``` returns ```AggregationData``` which are type of ```SumData | CountData | LastValueData | DistributionData;```